css/page-styles/wishlist.css

```diff
@@ -0,0 +1,32 @@
+@import url(../layouts/navbar.css);
+@import url(../layouts/footer.css);
+@import url(../utilities/variables.css);
+
+nav{
+    font-family: var(--acme-font);
+}
+html{
+    background-color: var(--primary-transparent);
+}
+body{
+    font-family: var(--lato-font);
+}
+.container{
+    justify-content: center;
+    width: 100%;
+    gap: 1rem;
+    flex-wrap: wrap;
+    
+}
+#heart{
```

Comment:
`id` in CSS is not a good practice. Try to use classes only in CSS

Reply:
Thanks for your Review. I have made these changes in new commit, please check it.